src/json/test.html

```diff
@@ -1,22 +1,41 @@
-<!doctype html>
-<title>Steal json tests</title>
-<link rel="stylesheet" type="text/css" href="../../node_modules/qunitjs/qunit/qunit.css"/>
-
-<h1 id="qunit-header">Steal Unit Test Suite</h1>
-
-<h2 id="qunit-banner"></h2>
-<div id="qunit-testrunner-toolbar"></div>
-<h2 id="qunit-userAgent"></h2>
-<ol id="qunit-tests"></ol>
-<div id="qunit-test-area"></div>
-
-
-<script src="../../node_modules/qunitjs/qunit/qunit.js"></script>
-<script src="../../steal.js" base-url="../../" config-main="@empty" main="@empty"></script>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>json extension tests</title>
+</head>
+<body>
+<div id="qunit"></div>
+<div id="qunit-fixture"></div>
 <script>
-	QUnit.config.autostart = false;
-
-	steal.import("src/json/json_test").then(function(){
-		QUnit.start();
+	var steal = {
+		paths: {
+			"steal-qunit": "node_modules/steal-qunit/steal-qunit.js"
+		},
+		map: {
+			"qunitjs": "node_modules/qunitjs"
+		},
+		meta: {
+			"node_modules/qunitjs/qunit/qunit": {
+				"format": "global",
+				"exports": "QUnit"
+			}
+		},
+		ext: {
+			"css": "node_modules/steal-css/css.js"
+		}
+	}
+</script>
+<script
+        src="../../../steal.js"
+        main="src/json/test/"
+        config-main="../../../package.json!npm"
+>
+</script>
+<script>
+	steal.done().then(function () {
+		QUnit.config.hidepassed = true;
```

Comment:
I like to see the tests as they run, personally. Some times I want to rerun tests even if they pass and this option prevents them from even showing up in the list.